blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js

```diff
@@ -743,9 +749,32 @@ export default function SimulationScreen ({ open, close }) {
                           : <div />
                       }
                       </>
-                    : (result.isGraph === 'true') ? <span>{typography1}</span> : <span />
+                    : (result.isGraph === 'false') ? <span>{typography1}</span> : <span />
+                }
+                {/* Diplay of Simulation parameter Not present */}
+                {
+                  (error === 1)
```

Comment:
```suggestion
                  (error !== '')
```
change to check for nonempty string